spec/system/my_learning_spec.rb

```diff
@@ -75,4 +75,40 @@
       end
     end
   end
+
+  context 'when a user has completed all modules except upcoming modules' do
```

Comment:
You can't start upcoming modules because they haven't been released. Is this clear from the context description. @David-Feetenby ?
In the `test` env `delta` will always be in draft mode.